src/test/java/api/PrecedingSucceedingTitlesApiExamples.java

```diff
@@ -0,0 +1,496 @@
+                                                    IndividualResource createdInstance) throws MalformedURLException,
+    InterruptedException, ExecutionException, TimeoutException {
+
+    Response response = instancesClient.getById(relatedInstance.getId());
+    JsonArray succeedingTitles = response.getJson().getJsonArray("succeedingTitles");
+    assertPrecedingSucceedingTitles(succeedingTitles.getJsonObject(0), createdInstance.getJson(),
+      relatedInstance.getId().toString(), createdInstance.getId().toString());
+  }
+
+  private JsonObject createConnectedPrecedingTitle(String id, String precedingInstanceId) {
+    return new JsonObject()
+      .put("id", id)
+      .put("precedingInstanceId", precedingInstanceId);
+  }
+
+  private JsonObject createConnectedSucceedingTitle(String id, String succeedingInstanceId) {
+    return new JsonObject()
+      .put("id", id)
+      .put("succeedingInstanceId", succeedingInstanceId);
+  }
+
+  private JsonObject getRecordById(JsonArray collection, String id) {
+    return collection.stream()
+      .map(index -> (JsonObject) index)
+      .filter(request -> StringUtils.equals(request.getString("id"), id))
+      .findFirst().get();
+  }
+
+  private JsonArray getUnconnectedPrecedingSucceedingTitle(String id1,
+                                                           String id2) {
+    JsonObject precedingSucceedingTitle1 = createSemanticWebUnconnectedTitle(id1);
+    JsonObject precedingSucceedingTitle2 = createOpenBibliographyUnconnectedTitle(id2);
+
+    JsonArray precedingSucceedingTitles = new JsonArray();
+    precedingSucceedingTitles.add(precedingSucceedingTitle1);
+    precedingSucceedingTitles.add(precedingSucceedingTitle2);
+    return precedingSucceedingTitles;
+  }
+
+  private JsonObject createOpenBibliographyUnconnectedTitle(String precedingSucceedingTitleId2) {
+    return new JsonObject()
+      .put("id", precedingSucceedingTitleId2)
+      .put("title", "Open Bibliography for Science, Technology and Medicine")
+      .put("hrid", "inst000000000555")
+      .put("identifiers", new JsonArray().add(
+        new JsonObject()
+          .put("identifierTypeId", "8261054f-be78-422d-bd51-4ed9f33c8012")
+          .put("value", "0662012103")));
+  }
+
+  private JsonObject createSemanticWebUnconnectedTitle(String precedingSucceedingTitleId1) {
+    return new JsonObject()
+      .put("id", precedingSucceedingTitleId1)
+      .put("title", "A semantic web prime")
+      .put("hrid", "inst000000000022")
+      .put("identifiers", new JsonArray().add(
+        new JsonObject()
+          .put("identifierTypeId", "8261054f-be78-422d-bd51-4ed9f33c3422")
+          .put("value", "0262012103")));
+  }
+
+  private void assertPrecedingSucceedingTitles(JsonObject actualPrecedingTitle, JsonObject expected, String precedingInstanceId,
+                                               String succeedingInstanceId) {
```

Comment:
Please can this be formatted with standard indentation

And please can more of the parameters from the first line be included in the second line, as I believe it is quite long and likely over the 80 character limit that FOLIO uses